OutOfSchool/OutOfSchool.WebApi/Controllers/V1/BaseMementoController.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using OutOfSchool.BusinessLogic.Common;
+using OutOfSchool.BusinessLogic.Services.Memento.Interfaces;
+
+namespace OutOfSchool.WebApi.Controllers.V1;
+
+/// <summary>Base Controller with operations for storing data in cache.</summary>
+/// <typeparam name="T">T is the entity type that should be stored in the cache.</typeparam>
+public abstract class BaseMementoController<T> : ControllerBase
```

Comment:
Please rename it!

Reply:
Is the name MementoController<T> appropriate?